### ISSUE TITLE
Missing key maybe? KeyError: 'ARVOPAPERI'

### ISSUE THREAD

**Issue by jhellen** [koodaamo/banking.statements.osuuspankki#3]
I got the following error when converting. Sounds just like it would be missing this key ARVOPAPERI. When I edited that word to something else in the CSV file it converted just fine.

```
Traceback (most recent call last):
  File "/usr/local/bin/ofxstatement", line 9, in <module>
    load_entry_point('ofxstatement==0.6.0', 'console_scripts', 'ofxstatement')()
  File "/usr/local/lib/python3.5/dist-packages/ofxstatement-0.6.0-py3.5.egg/ofxstatement/tool.py", line 150, in run
  File "/usr/local/lib/python3.5/dist-packages/ofxstatement-0.6.0-py3.5.egg/ofxstatement/tool.py", line 128, in convert
  File "/usr/local/lib/python3.5/dist-packages/ofxstatement-0.6.0-py3.5.egg/ofxstatement/parser.py", line 28, in parse
  File "/usr/local/lib/python3.5/dist-packages/banking.statements.osuuspankki-1.3.3rc0-py3.5.egg/banking/statements/osuuspankki/parser.py", line 77, in parse_record
    line[4] = TRANSACTION_TYPES[line[4]]
KeyError: 'ARVOPAPERI'
```

**Comment by petri** [comment]
Yes you are right. Thanks for reporting. What kind of transaction is this? For example, is it income from selling some stocks or bonds ("osakemyyntitulo")? Or income from dividends ("osinko")? Or something else?

 I am asking because if you take a look at the code (https://github.com/koodaamo/banking.statements.osuuspankki/blob/master/banking/statements/osuuspankki/__init__.py), you will see that the transaction type has to be mapped to an OFX transaction type.

So to fix this, I need to know what this transaction type represents.

**Comment by jhellen** [comment]
It is from buying funds (rahasto).

**Comment by petri** [comment]
Fixed in a0e5484. If you think some other mapping is better, let me know.
